Sync/VirtuousService.cs

```diff
@@ -35,5 +36,36 @@ public async Task<PagedResult<AbbreviatedContact>> GetContactsAsync(int skip, in
             var response = await _restClient.PostAsync<PagedResult<AbbreviatedContact>>(request);
             return response;
         }
+
+        public async Task<PagedResult<AbbreviatedContact>> GetContactsAsync(int skip, int take, string stateAbbreviation)
```

Comment:
I went with an overload here instead of modifying the existing method so that the old method to get everything could still be used as is. In a real repo I would imagine that method could have multiple callers. Here it was useful to have it around to load test my solution for Part 3 of the assesment